extensions/provisioning/verify.go

```diff
@@ -103,6 +103,65 @@ func VerifyRKE1Cluster(t *testing.T, client *rancher.Client, clustersConfig *clu
 	}
 }
 
+// VerifyRKE1ClusterWithTimeout validates that the RKE1 cluster and its resources are in a good state within the custom timeout, matching a given config.
+func VerifyRKE1ClusterWithTimeout(t *testing.T, client *rancher.Client, clustersConfig *clusters.ClusterConfig, cluster *management.Cluster, timeout *int64) {
```

Comment:
These funcs are going to be moved out to rancher/rancher actions. Do you just want to create local ones in your repo?

Reply:
Is this entire file being removed from shepherd, or just all the "Verify" funcs?